polyaxon_cli/run/docker.py

```diff
@@ -124,7 +127,7 @@ def _create_docker_build(build_job, build_config, project):
             raise PolyaxonShouldExitError('')
 
         image_tag = hash_value(rendered_dockerfile)
-        if POLYAXON_NO_OP_KEY not in os.environ:
+        if not settings.IS_OFFLINE:
```

Comment:
I think it's better to use `if 'POLYAXON_IS_OFFLINE' in os.environ:` similar to [line](https://github.com/polyaxon/polyaxon-cli/pull/36/files#diff-4bc442d573f1f53e5ed495733f227038R48), without having to depend on the `settings` coming from the client.

Reply:
Fixed this.